shared/js/background/chrome-events.es6.js

```diff
@@ -124,6 +124,22 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
         ATB.setAtbValuesFromSuccessPage(req.atb)
     }
 
+    // listen for messages from content scripts injected into frames
+    // on specific domains. Respond with list of blocked requests.
+    if (req.hideElements) {
+        let requestTab = tabManager.get({tabId: sender.tab.id})
+        if (requestTab.parentEntity === 'Oath') {
+            if (req.frame === 'main') {
+                chrome.tabs.sendMessage(sender.tab.id, {type: 'blockedRequests', blockedRequests: requestTab.framesBlocked, frame: 'main'}, {frameId: sender.frameId})
```

Comment:
This might be why I'm sometimes seeing blocked ads that are not hidden.

Reply:
This is a good idea. I should be able to switch things around here.

Reply:
Since there's some time pressure to get this in the upcoming release, I'll address this in a future PR. I agree it would improve the approach, but for now it's 'good enough'.